src/services/config.js

```diff
@@ -0,0 +1,103 @@
+const common = require('@constants/common')
+const organizationExtensionsQueries = require('@database/queries/organizationExtensions')
+const _ = require('lodash')
+const entites = require('@database/queries/entities')
+const entityType = require('@database/queries/entityType')
+const httpStatusCode = require('@generics/http-status')
+const responses = require('@helpers/responses')
+const { Op } = require('sequelize')
+module.exports = class configsHelper {
+	/**
+	 * List Configs.
+	 * @method
+	 * @name list
+	 * @param {Integer} organization_id
+	 * @param {String} id -  id.
+	 * @returns {JSON} - List of configs as response.
+	 */
+
+	static async list(organization_id) {
+		try {
+			let orgExtenstionData = {}
+			let configData = []
+			// define filter
+			const filter = {
+				organization_id,
+			}
+			// attributes to fetch from organisation Extenstion
+			const attributes = process.env.INSTANCE_LEVEL_CONFIG_ATTRIBUTES.split(',')
+
+			const entity_type_and_entities = await entityType.findUserEntityTypesAndEntities({
+				organization_id: organization_id,
+				value: 'resources',
+			})
+			// fetch the current list of resources
+			const resourceList = entity_type_and_entities[0].entities
```

Comment:
@adithyadinesh0412 use find one instead of the above function, so we don't need to take 1st element of area, the issue is array can be empty then it will throw an error